content/docs/guides/continuous-delivery/teamcity.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "JetBrains TeamCity"
+meta_desc: "This page provides an overview of how to use Pulumi with JetBrains TeamCity."
+
+menu:
+    userguides:
+        parent: cont_delivery
+        weight: 1
+
+aliases:
+- /docs/reference/cd-jetbrains-teamcity/
+- /docs/console/continuous-delivery/jetbrains-teamcity/
```

Comment:
@stack72, if this is a brand new page, these aliases should be deleted. The aliases are only necessary for existing pages that have changed paths within the site.